cep-checker/go.mod

```
@@ -3,124 +3,134 @@ module github.com/cybozu/neco-container/cep-checker
 go 1.24.1
```

You may be able to update golang to this version.
https://github.com/cybozu/neco-containers/pkgs/container/golang
```suggestion
go 1.24.5
```